internal/pkg/filesystem/localfs/localfs.go

```diff
@@ -41,3 +46,53 @@ func (fs *LocalFs) BasePath() string {
 func (fs *LocalFs) Walk(root string, walkFn filepath.WalkFunc) error {
 	return fs.utils.Walk(root, walkFn)
 }
+
+func (fs *LocalFs) ReadDir(path string) ([]os.FileInfo, error) {
+	return fs.utils.ReadDir(path)
+}
+
+// FindProjectDir -> working dir or its parent that contains ".keboola" metadata dir.
+func FindProjectDir(logger *zap.SugaredLogger, workingDir string) (string, error) {
```

Comment:
Pridana metoda na najedenie `projectDir`.
Je to presunute do filesystem, kedze to dava zmysel iba pri `localfs`.
Ak mas nejaky `workingDir` ale `projectDir` moze byt nejaky jeho nadradeny adresar, ... tj. si aktualne v niektorej pod-zlozke, ale aj tak potrebujes aby `kbc` prikaz fungoval s `projectDir`.

Pri `memoryfs` to nedava zmysel.